app/components/image-selector.cjsx

```diff
@@ -1,6 +1,6 @@
 React = require 'react'
 LoadingIndicator = require './loading-indicator'
-toBlob = require 'data-uri-to-blob'
+toBlob = require 'data-uri-to-blob' if window.navigator?
```

Comment:
Yeah I added a window-shim, so the actual test for things that won't load at all in the server environment ends up being `if window.navigator?`